Gemfile

```diff
@@ -1,13 +1,15 @@
 source 'https://rubygems.org'
 
-ruby '2.5.3'
-gem 'puma'
+ruby '2.7.6'
+gem 'puma', '4.3.12'
```

Comment:
Specified puma version to a minor upgrade. By default, it upgraded to the latest, which borked things.